lib/y2lan_restart_common.pm

```diff
@@ -55,14 +75,30 @@ sub initialize_y2lan
     assert_script_run '> journal.log';    # clear journal.log
 }
 
+=head2 open_network_settings
+
+ open_network_settings();
+
+Open yast2 lan module, expecting Overview tab and select first device.
+
+Accept warning for Networkmanager controls network device.
```

Comment:
I would include this sentence in the previous one.

Reply:
Accept warning for Networkmanager controls network device -> in which previous one?

Reply:
it is fine now I think.